### ISSUE TITLE
Feature: support -u 

### ISSUE THREAD

**Issue by joegaudet** [lujiajing1126/redis-cli#25]
At present doesn't seem to support the -u param.

**Comment by lkonzen-garupa** [comment]
I'm trying to use this, as described in the help ```rdcli --help``` but doesn't work.

Always return with error.

```cmd
(error) Ready check failed: NOAUTH Authentication required
```

But my url informed is on this format:

```cmd
redis://USER:PASSWORD@HOST_NAME:PORT
```

Whe i try with the same info, but using the ```-h```, ```-a``` and ```-p``` params, it works.

**Comment by lujiajing1126** [comment]
@joegaudet @lkonzen-garupa Thanks for the report!

It has been fixed in the new release `1.4.0`. Pls update and have a try.
